contributing.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to this Repo
+## What should I work on?
+- Take a look at our open issues, if you are new to this project, look for the ones with the
+`Good First Issue` tag.
+- Take a look at our projects, if you think there is something you would like
+to work on that contributes to the project, then feel free to work on it and make a PR!
+- You can always reach out to me ([hello@czhang.me](mailto:hello@czhang.me)) or any of the
+contributors, we can talk about what our next steps are.
+
+## Development Process
+1. If you have something you would like to work on, or you found a bug, please open an issue
+if there isn't one already. If the issue is releated to a certain project, please
+tag the project in the issue.
+1. If you would like to "own" a certain open issue, assign yourself to it.
```

Comment:
why are all these numbers 1

Reply:
in markdown it will just automatically increment, so if you wanna add an item in between, u don't have to change all the numbers

Reply:
😮